cmd/api/src/api/v2/file_uploads.go

```diff
@@ -130,15 +131,17 @@ func (s Resources) ProcessFileUpload(response http.ResponseWriter, request *http
 		fileUploadJobIdString = mux.Vars(request)[FileUploadJobIdPathParameterName]
 	)
 
-	if fileUploadJobID, err := strconv.Atoi(fileUploadJobIdString); err != nil {
+	if !api.HeaderMatches(headers.ContentType.String(), mediatypes.ApplicationJson.String(), request.Header) && !api.HeaderMatches(headers.ContentType.String(), mediatypes.ApplicationZip.String(), request.Header) {
```

Comment:
There's a constant for accepted types `bhce/cmd/api/src/api/v2/file_uploads.go`. I think it would be a good idea to rely on that constant here if possible.